### ISSUE TITLE
The block cursor keep jumping to the previous position.

### ISSUE THREAD

**Issue by duytrung** [microsoft/vscode#223462]
Does this issue occur when all extensions are disabled?: No

- VS Code Version: 1.92 Insider
- OS Version: Windows 11

Steps to Reproduce:

1. Start Extension Bisect to isoloate the bad extensions
2. Finished with bisect but the problem is still persist.

![image](https://github.com/user-attachments/assets/0dfb474c-316f-4cd1-acf6-eb1dd680dd7b)

Description of the problem:
I started with VScode Insider, and start typing, but the cursor keeps jump to the previous position and it is so annoying. For example, I type "pro" it will appears as pr[o] where [o] is a block cursor with the letter "o" nested in.

The block cursor keep jumping to the previous position.

![image](https://github.com/user-attachments/assets/12469a47-74bc-484f-9f51-beea6e6b436d)

I tried with the "Code - Insiders.exe" --list-extensions but not working.

Thanks for considering the issue

**Comment by duytrung** [comment]
Since I tried to list all extension with the command ["Code - Insiders.exe" --list-extensions > plugs.txt] did not really work, I would like to attach the plugins json in data folder here.

[extensions.json](https://github.com/user-attachments/files/16358761/extensions.json)

I have tried to isolate the trouble plugins but it was so strange that the bisection results was the same as above.
Did I miss anything?
